core/client/app/routes/posts.js

```diff
@@ -18,7 +18,7 @@ export default AuthenticatedRoute.extend(ShortcutsRoute, PaginationRouteMixin, {
 
         return this.get('session.user').then(function (user) {
             if (user.get('isAuthor')) {
-                paginationSettings.author = user.get('slug');
+                paginationSettings.filter += `+author:${user.get('slug')}`;
```

Comment:
`.filter` is undefined at this point, there are two options:

1. Set `filter` in the default `paginationSettings` to `''`
2. Change this line to ```paginationSettings.filter = `author:${user.get('slug')}```` - is the `+` in front of `author` always needed or only if it's adding to an existing filter?

I'd suggest option 1 is probably best as I expect the filter parameter will be used a lot more once we have the filtering UI in place.